spec/lib/appsignal/integrations/http_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+if DependencyHelper.http_present?
+  require "appsignal/integrations/http"
+
+  describe Appsignal::Integrations::HttpIntegration do
+    around do |example|
+      keep_transactions { example.run }
+    end
+
+    before :context do
+      start_agent
+    end
+
+    it "should instrument a HTTP request" do
+      stub_request(:get, "http://www.google.com/")
+
+      expect { HTTP.get("http://www.google.com") }
+        .to change { created_transactions.length }.by(1)
+
+      expect(last_transaction).to be_completed
+      expect(last_transaction.to_h).to include("namespace" => Appsignal::Transaction::HTTP_REQUEST)
```

Comment:
I would add to these specs an assertion if the event has been created by `Appsignal.instrument`. Like we do here in the [que specs](https://github.com/appsignal/appsignal-ruby/blob/220f83efd98db7102a3a38f12f1dc939e0dfa080/spec/lib/appsignal/integrations/que_spec.rb#L64-L77). I think you can limit it to just the `name`, `title`, `body` and `body_format` keys (even if `body` is just an empty String).

```suggestion
      expect(transaction.to_h["events"].first).to include(
        "body" => "",
        "body_format" => Appsignal::EventFormatter::DEFAULT,
        "name" => "request.http_rb",
        "title" => "GET http://www.google.com"
      )
```